crates/routes/src/utils/code_migrations.rs

```diff
@@ -1,474 +0,0 @@
-    if !site.public_key.is_empty() {
-      return Ok(());
-    }
-    let key_pair = generate_actor_keypair()?;
-    let ap_id = Url::parse(protocol_and_hostname)?;
-    let site_form = SiteUpdateForm {
-      ap_id: Some(ap_id.clone().into()),
-      last_refreshed_at: Some(Utc::now()),
-      inbox_url: Some(generate_inbox_url()?),
-      private_key: Some(Some(key_pair.private_key)),
-      public_key: Some(key_pair.public_key),
-      ..Default::default()
-    };
-    Site::update(pool, site.id, &site_form).await?;
-  }
-  Ok(())
-}
-
-/// Fix for bug #2347, which can result in community/person public keys being overwritten with
-/// empty string when the database value is updated. We go through all actors, and if the public
-/// key field is empty, generate a new keypair. It would be possible to regenerate only the pubkey,
-/// but thats more complicated and has no benefit, as federation is already broken for these actors.
-/// https://github.com/LemmyNet/lemmy/issues/2347
-async fn regenerate_public_keys_2022_07_05(pool: &mut DbPool<'_>) -> LemmyResult<()> {
-  let conn = &mut get_conn(pool).await?;
-  info!("Running regenerate_public_keys_2022_07_05");
-
-  {
-    // update communities with empty pubkey
-    use lemmy_db_schema_file::schema::community::dsl::{community, local, public_key};
-    let communities: Vec<Community> = community
-      .filter(local.eq(true))
-      .filter(public_key.eq(""))
-      .load::<Community>(conn)
-      .await?;
-    for community_ in communities {
-      info!(
-        "local community {} has empty public key field, regenerating key",
-        community_.name
-      );
-      let key_pair = generate_actor_keypair()?;
-      let form = CommunityUpdateForm {
-        public_key: Some(key_pair.public_key),
-        private_key: Some(Some(key_pair.private_key)),
-        ..Default::default()
-      };
-      Community::update(&mut conn.into(), community_.id, &form).await?;
-    }
-  }
-
-  {
-    // update persons with empty pubkey
-    use lemmy_db_schema_file::schema::person::dsl::{local, person, public_key};
-    let persons = person
-      .filter(local.eq(true))
-      .filter(public_key.eq(""))
-      .load::<Person>(conn)
-      .await?;
-    for person_ in persons {
-      info!(
-        "local user {} has empty public key field, regenerating key",
-        person_.name
-      );
-      let key_pair = generate_actor_keypair()?;
-      let form = PersonUpdateForm {
-        public_key: Some(key_pair.public_key),
-        private_key: Some(Some(key_pair.private_key)),
-        ..Default::default()
-      };
-      Person::update(pool, person_.id, &form).await?;
-    }
-  }
-  Ok(())
-}
-
-/// This ensures that your local site is initialized and exists.
-///
-/// If a site already exists, the DB migration should generate a local_site row.
-/// This will only be run for brand new sites.
-async fn initialize_local_site_2022_10_10(
```

Comment:
CI probably failing because a local site still needs to be set up? Or the rate limit considerations below are failing.

Reply:
Right, fixed by adding this function back.